web/public/index.html

```diff
@@ -49,5 +49,13 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <link rel="stylesheet/less" type="text/css" href="%PUBLIC_URL%/color.less" />
+    <script>
+      less = {
+        async: false,
```

Comment:
Remove this

Reply:
it can't be remove, we need import the less file here and set parameter of less.